src/parser/jobs/ast/modules/Divination.tsx

```diff
@@ -0,0 +1,297 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {RotationTable} from 'components/ui/RotationTable'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Actor, Actors} from 'parser/core/modules/Actors'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, Suggestion} from 'parser/core/modules/Suggestions'
+import {Timeline} from 'parser/core/modules/Timeline'
+import React, {Fragment} from 'react'
+import {Icon, Message} from 'semantic-ui-react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+const BASE_GCDS_PER_WINDOW = 6
+const PLAYERS_BUFFED_TARGET = 8
+
+interface DIVINATION_Window {
+	start: number
+	end?: number
+	source: Actor['id']
+
+	rotation: Array<Events['action']>
+	gcdCount: number
+	trailingGcdEvent?: Events['action']
+	playersBuffed: Array<Actor['id']>
+	overlap: boolean
+}
+
+// in this module we only want to track Divination windows opened by
+// the character selected for analysis. windows that clip into
+// AST Divination will be marked.
+// Used DNC Technicalities as basis for this module. Rewritten from previous module for consistency purposes
+export default class Divination extends Analyser {
```

Comment:
No default export.
```suggestion
export class Divination extends Analyser {
```

Reply:
I get this error when I do that
![image](https://user-images.githubusercontent.com/89435408/145928480-c4737c5f-e48b-414d-8b6d-8fd843cc0c8d.png)

Reply:
pls ignore this -- resolved in other conversation